rfcs/0019-data-structures/0019-data-structures.md

```diff
@@ -0,0 +1,255 @@
+---
+Number: "0000"
```

Comment:
Number.

Reply:
Looks like we need a script to check such errors via CI.